deploy-as-code/helm/charts/digit-works/backend/ifms-adapter/values.yaml

```diff
@@ -40,3 +55,58 @@ env: |
   - name: PAYMENT_STATUS_KAFKA_TOPIC
```

Comment:
@shubhang-eGov change this value in both application.properties as well as here to be more informative. It needs to be PAYMENT_CREATE_TOPIC.

Reply:
done